packages/mantine/src/r4b/hooks/use-fhir-form.tsx

```diff
@@ -35,14 +35,15 @@ export function useFhirForm<
   Values = Record<string, unknown>,
   TransformValues extends _TransformValues<Values> = (values: Values) => Values,
 >(
-  args?: UseFormInput<Values, TransformValues>,
+  args?: UseFormInput<Values, any>,
```

Comment:
@julienblin you might have a more clever solution to fix that one 🤔

Reply:
Yeah it's a problem of transitivity of generic types.
I would advise against altering the `args` though, as it is the right type.

Here is what I suggest:

```typescript
export function useFhirForm<
  Values = Record<string, unknown>,
  TransformValues extends _TransformValues<Values> = (values: Values) => Values,
>(
  args?: UseFormInput<Values, TransformValues>,
): UseFhirFormReturnType<Values, TransformValues> {
  const { transformValues, ...remainingArgs } = args || {};

  const form = useForm<Values, TransformValues>({
    ...remainingArgs,

    transformValues: ((values: Values) => {
      cleanFhirValues(values);
      return transformValues ? transformValues(values) : values;
    }) as any,
  });
  return {
```